src/consensus/aft/impl/execution.h

```diff
@@ -37,16 +37,20 @@ namespace aft
     virtual kv::Version execute_request(
       std::unique_ptr<RequestMessage> request,
       bool is_create_request,
-      std::shared_ptr<aft::RequestTracker> request_tracker = nullptr) = 0;
+      kv::Consensus::SeqNo last_idx = -1,
```

Comment:
`max_conflict_version` is the version the store must be at least at, for the request to execute correctly, as I understand it. It's not clear to me what `last_idx` is at this point?